### PR TITLE
Hello Microverse: First pull request

### PR DESCRIPTION
**Pull request by dewslyse**
- Updated README.md

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,97 +1,72 @@
-# HTML & CSS3 Course
+![](https://img.shields.io/badge/Microverse-blueviolet)
```
Comment (mateo951) [inline README.md:1]: Awesome readme file 🤓 Great work!

**Review by mateo951 — CHANGES_REQUESTED** [review]
### Hi @dewslyse,

Welcome to Microverse and congrats on making your first project 💪😎 

📝👌 **Your project highlights** 🔥😎 
- Nice README file 🚀
- Great use of GitHub Flow 🤯 (Nice name for your feature branch 🔥)
- Good PR's title

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
- [ ]  Kindly add node_modules to the .gitignore file so we practice for the near future when we work with dependencies 🧐

image


- [ ] Please be sure to make all of your commits in your feature branch on future occasions so that all new implementations and changes/additions of files are visible 😉


## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)_

File: README.md (hunk 1 above)
Comment (dewslyse) [reply README.md:1]: Thank you.

**Review by Deepakdanger — APPROVED** [review]
Hi @dewslyse, 

### Status: Approved ✔️

Your project is complete!  There is nothing else to say other than... it's time to merge it  🚢 
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**